docs/release-notes.md

```diff
@@ -9,17 +9,52 @@ description: >
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.8.2`. For
+The latest tagged release is `2.8.3`. For
 installation instructions see our
 [Install](/docs/install/) page.
 
-The latest CLI release is `2.8.2`, available from
+The latest CLI release is `2.8.3`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
+## 2.8.3 - Release 2022-09-14
+
+### New
+
+Data Plane
+
+* Add support for bottlerocket
+* Check whether the block device directory (usually /var/lib/storageos/volumes) is mounted with `nodev` and/or `ro`, and if so attempt to remount the filesystem `rw` and with `dev` access
```

Comment:
Change to:
Check whether the block device directory (usually /var/lib/storageos/volumes) supports creation of devices, and enable it if it is not already.